src/SprykerEco/Service/ComputopApi/ComputopApiServiceInterface.php

```diff
@@ -133,6 +133,7 @@ public function getBlowfishDecryptedValue($cipherText, $length, $password);
 
     /**
      * Specification:
+     * - Requires `QuoteTransfer.TotalsTransfer.hash` and `QuoteTransfer.CustomerTransfer` properties
```

Comment:
```suggestion
     * - Requires `QuoteTransfer.totals.hash` and `QuoteTransfer.customer` properties
```

please fix all similar cases

Reply:
Fixed